uberfire-workbench/uberfire-workbench-processors-tests/src/main/java/org/uberfire/annotations/processors/AbstractProcessorTest.java

```diff
@@ -98,6 +98,7 @@
      */
     public String getExpectedSourceCode(final String compilationUnit) throws FileNotFoundException {
         StringBuilder sb = new StringBuilder();
+        String expectedCode=null;
```

Comment:
could you please make sure to use our code style?

Reply:
reformatted it